cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,25 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+    url = '/#/articles';
+    
+    get editArticleBtn() {
+        return cy.getByDataCy('edit-article-btn');
+    }
+
+    get deleteArticleBtn() {
+        return cy.getByDataCy('delete-article-btn')
```

Comment:
There is a missing semicolon at the end of the `return` statement in the `deleteArticleBtn` getter method. Consider adding a semicolon to maintain consistency and prevent potential issues.